src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -580,7 +585,7 @@ protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, b
                 return Utils.MarshalInvoke(MakeGetter<int>, type, input, iinfo, srcTensorGetters, activeOutputColNames, outputCache);
             }
 
-            private Delegate MakeGetter<T>(DataViewRow input, int iinfo, ITensorValueGetter[] srcTensorGetters, string[] activeOutputColNames, OutputCache outputCache)
+            private Delegate MakeGetter<T>(DataViewRow input, int iinfo, ITensorValueGetter[] srcTensorGetters, string[] activeOutputColNames, OutputCache outputCache) where T:unmanaged
             {
```

Comment:
refactor code for unmanged as done in ImageClassificationTranform.

Reply:
I did the refactor and looks like one of the tests was hitting this code path for string. This means that the original MakeGetter was not erroring out for the type string even after 'where T: unmanaged' was specified. Do we still want to refactor the code?

---
In reply to: [324894953](https://github.com/dotnet/machinelearning/pull/4205#discussion_r324894953) [](ancestors = 324894953)